src/pages/index.tsx

```diff
@@ -26,81 +29,24 @@ export default function Home() {
 
       <Ecosystem />
 
-      <main className={cn(styles.page)}>
-        <h2>Экосистема Екатеринбурга</h2>
-
-        <h3>Что происходит</h3>
-
-        <p>
-          Сегодня город&nbsp;&mdash; это не&nbsp;просто место, чтобы жить жизнь,
-          а&nbsp;невероятно сложная инфраструктура услуг, функций, событий,
-          сценариев и взаимодействий, которые определяют комфорт и&nbsp;качество
-          жизни. Разобраться во&nbsp;всём этом, мягко говоря, сложно.
-        </p>
-
-        <p>
-          Поэтому мы&nbsp;планируем создать цифровую экосистему Екатеринбурга
-          &mdash; объединение всех городских сфер в&nbsp;одном месте, чтобы
-          упростить и автоматизировать жизнь горожан, бизнеса
-          и&nbsp;администрации.
-        </p>
-
-        <h3>Почему здесь</h3>
-
-        <p>
-          Потому что Екатеринбург&nbsp;&mdash; город энтузиастов. Мы&nbsp;горим
-          своим делом. Мы объединяемся, чтобы свернуть горы,
-          а&nbsp;не&nbsp;конкурируем ради прибыли. Мы &mdash; сообщество.
-        </p>
-
-        <h3>Что уже есть и&nbsp;что в&nbsp;планах</h3>
-
-        <p>
-          Уже есть 10&nbsp;проектов и&nbsp;заряженная команда&nbsp;&mdash;
-          зацените, какой у&nbsp;нас
-          <a href="https://github.com/ekaterinburgdev/" target="_blank">
-            гитхаб
-          </a>
-          . В&nbsp;ближайшие месяцы планируются большие анонсы всех проектов и
-          разработка новых.
-        </p>
+      <MainText
+        projects={<Iframe src="https://projects.ekaterinburg.dev/widget" />}
+        team={<Team />}
```

Comment:
_my rebase updates:_

I give `<Team>` amd `<Iframe>` as props. But if you want, we can delete `<MainPage>`